ci/travis_jobs/.docker_password

```diff
@@ -0,0 +1 @@
+Qc!pher9
```

Comment:
What is this?

Reply:
Something that needs to be removed.  I tried catting a clear text password but was still getting permission denied because John hadn't updated the permissions